site/src/scaffold/index.scss

```diff
@@ -1,3 +1,4 @@
 @import '@microsoft/atlas-css/src/index.scss';
 @import './styles/layout.scss';
+@import './styles/misc.scss';
```

Comment:
Just getting some PTSD regarding a previous misc.scss file. Can we keep them specific for now at least?

@import './styles/height.scss';
@import './styles/overflow.scss';